src/ui/public/styles/base.less

```diff
@@ -157,6 +157,16 @@ a {
       .button-group > :last-child {
         border-radius: 0;
       }
+
+      button.is-kbn-top-nav-button-disabled {
```

Comment:
Is it possible to simplify this to just the class `.is-kbn-top-nav-button-disabled`?